Sources/VLCPlaybackController.h

```diff
@@ -13,7 +13,8 @@
 
 #import "VLCEqualizerView.h"
 
-extern NSString *const VLCPlaybackControllerPlaybackDidStart;
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Shouldn't the nonnull be another commit which would come before this one? I feel that it's kind of wrong that this commit touches `VLCRendererDiscovererManager`.